support.md

```diff
@@ -0,0 +1,44 @@
+# Support
+
+This integration is built as open source which you are free to use and customise for your implementation. Amplience supports this integration for the expected behaviour and roles between the 3 key parts of the integration:
```

Comment:
The second sentence is a bit hard to parse.

Reply:
Split into two?